autoinstall/Ubuntu/Desktop/ubuntu.seed

```diff
@@ -42,14 +42,15 @@ d-i passwd/username string {{ new_user }}
 #d-i passwd/user-password-again password insecure
 # or encrypted using an MD5 hash.
 #d-i passwd/user-password-crypted password [MD5 hash]
-d-i passwd/user-password password {{ vm_password }}
-d-i passwd/user-password-again password {{ vm_password }}
+#d-i passwd/user-password password {{ vm_password }}
+#d-i passwd/user-password-again password {{ vm_password }}
+d-i passwd/user-password-crypted password {{ vm_password_md5 }}
 d-i passwd/auto-login boolean true
 d-i user-setup/encrypt-home boolean false
 d-i passwd/root-login boolean true
-d-i passwd/root-password password {{ vm_password }}
-d-i passwd/root-password-again password {{ vm_password }}
-#d-i passwd/root-password-crypted password {{ vm_password_hash }}
+#d-i passwd/root-password password {{ vm_password }}
+#d-i passwd/root-password-again password {{ vm_password }}
```

Comment:
Please remove these two lines directly instead of commenting them out.

Reply:
Thanks. Done